pkg/commands/publisher.go

```diff
@@ -41,6 +41,11 @@ func qualifyLocalImport(importpath string) (string, error) {
 	return pkgs[0].PkgPath, nil
 }
 
+// PublishImages publishes images
+func PublishImages(ctx context.Context, importpaths []string, pub publish.Interface, b build.Interface) (map[string]name.Reference, error) {
```

Comment:
Is there a reason to prefer exporting this new wrapper method, versus exporting `publishImages` directly?

Reply:
I think it makes the diff smaller, which is nice. I kind of like it :P

Reply:
I'd be okay with it as a specific diff-shrinking measure that we intend to clean up with a followup PR.

Reply:
Yes, it was just to keep the diff small.